src/unitxt/metrics.py

```diff
@@ -1430,3 +1447,27 @@ def _compute(
             for k in self.k_list:
                 result[self.score_name(measure_name, k)] = measure_array[min(k, max_k)]
         return result
+
+
+class KPA(NER):
```

Comment:
@yoavkatz suggestion: can we modify `F1MultiLabel` such that the grouping functions also have access to the additional_inputs passed to compute? and then we can directly group by an input field, instead of encoding that field in a tuple, to match the NER format?